src/query/service/src/interpreters/interpreter_insert.rs

```diff
@@ -262,6 +264,20 @@ impl Interpreter for InsertInterpreter {
             append_mode,
         )?;
 
+        let refresh_agg_index_desc = RefreshAggIndexDesc {
```

Comment:
Insert is not the main path for data ingest to Databend, COPY/REPLACE is.
Why we need to refresh the index after insert?

Reply:
Oh, there is a `get_enable_sync_refresh_aggregating_index()` settings already.
The name `enable_refresh_aggregating_index_after_write` is better?

Reply:
> Insert is not the main path for data ingest to Databend, COPY/REPLACE is.
Yep, I'll go on other ways after the insert is completed.

> Why we need to refresh the index after insert? The agg index refresh will run in the background service not the query.
Users may want the index generation in real time. cc @sundy-li

Reply:
This should not be a setting but another kind of aggregating index.

```
CREATE sync | [async]  AGGREGATING INDEX ....
```

Reply:
>  Users may want the index generation in real time.

That's right, we should provide this option to let users choose the suitable mode.